src/covid_model_seiir_pipeline/cli.py

```diff
@@ -62,7 +62,6 @@ def fit(run_metadata,
     fit_spec.data.location_set_version_id = locations_set_version_id
     fit_spec.data.location_set_file = location_set_file
     fit_spec.data.output_root = str(run_directory)
-    fit_spec.dump(run_directory / 'fit_specification.yaml')
```

Comment:
why was this removed? was the line redundant with another one?

Reply:
Consistency with other applications means that the main function should be responsible for everything besides the top level directory and the metadata.yaml file that lives within it.  Specifications are dumped in the main function now.